src/fsharp/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.nuspec

```diff
@@ -15,7 +15,6 @@
         <tags>$tags$</tags>
         <dependencies>
             <group targetFramework=".NETStandard1.6">
```

Comment:
if this package is for built `fsc`, target shouldnt be `netcoreapp1.0` instead of `netstandard1.6`?